spec/operations/story_operations/create_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+describe StoryOperations::Create do
+  let(:story_params) do
+    { title: 'Test Story', requested_by: user, state: 'unstarted', accepted_at: nil }
+  end
+
+  let!(:membership) { create(:membership) }
+  let(:user)        { membership.user }
+  let(:project)     { membership.project }
+  let(:story)       { project.stories.build(story_params) }
+
+  subject { -> { StoryOperations::Create.new.call(story: story, current_user: user) } }
+
+  context 'with valid params' do
+    it { expect { subject.call }.to change { Story.count } }
+    it { expect { subject.call }.to change { Changeset.count } }
```

Comment:
When changing the `count` for a given value, it best to specify how much was changed, this way leave some questions like, how many stories were created? So if this in fact created just a single story, you can test with `it { expect { subject.call }.to change { Story.count }.by(1) }`. Hope this helps

Reply:
yeah, I saw a lot of things that can be changed in those tests, til now I just moved them from the old `story_operations_spec.rb` file to each operation class file.
But I'll take a look at each test with this and your other comments and tips bellow in mind to make them better. ty :smile: